keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2256,11 +2123,16 @@ k8s.io/kubelet/pkg/apis/dra gRPC interface. It was inspired by
 [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md),
 with “volume” replaced by “resource” and volume specific parts removed.
 
-Versions v1alpha4 and v1beta1 are supported by kubelet. Both are identical.
-DRA drivers should implement both because support for v1alpha4 might get
-removed.
+Versions v1beta1 and v1 are supported by kubelet. Both are identical.
+DRA drivers should implement both because support for v1beta will get
```

Comment:
Should this be `v1beta1` (instead of just `v1beta`)?

Reply:
Yes.